### DISCUSSION TITLE
Toast/Notification Component

### DISCUSSION THREAD

**Discussion by ryall** [tailwindlabs/headlessui#1160] [Ideas]
Would be great to have a headless component to handle toast notifications, with emitter logic so notifications can be queued. 

Something like [react-toastify](https://github.com/fkhadra/react-toastify).

It would especially be useful as you have a [Notification component](https://tailwindui.com/components/application-ui/overlays/notifications) in TailwindUI, but currently has no proper logic attached.

**Comment by DannyDelott** [comment]
Take a look at this comment, there's `react-hot-toast` which works pretty seamlessly with the tailwind Notification component:

https://github.com/tailwindlabs/headlessui/discussions/439#discussioncomment-2161495
